spec/statement.dd

```diff
@@ -1902,6 +1902,40 @@ or when it exits due to exception unwinding. $(D scope(failure)) executes
 *NonEmptyOrScopeBlockStatement* when the scope exits due to exception unwinding.
 `scope(success)` executes *NonEmptyOrScopeBlockStatement* when the scope exits normally.)
 
+$(SPEC_RUNNABLE_EXAMPLE_RUN
+--------------
+import std.stdio;
+
+int foo()
+{
+    scope(exit) writeln("Inside foo()");
+    return bar();
+}
+
+int bar()
+{
+    writeln("Inside bar()");
+    return 0;
+}
+
+int main()
+{
+    foo();
+    return 0;
+}
+--------------
+)
+
+        writes:
+
+$(CONSOLE
+Inside bar()
+Inside foo()
+)
```

Comment:
Please move this example so that it is grouped with the others.

Reply:
Moved it, please check.